.schemastore/config.schema.json

```diff
@@ -2419,6 +2488,31 @@
               "default": 12
             }
           }
+        },
+        "legacyfandom": {
```

Comment:
Do we still need it? If I remember correctly at some point we were suppose to encourage users to change/rehash the password

Reply:
We didn't switch to new algorithm (at least on dev). New accounts are created with `legacyfandom`
```
select identity_credentials.config, identity_credentials.created_at from fandom_auth.identities
         inner join fandom_auth.identity_credentials on identities.id = identity_credentials.identity_id
    where identities.id='3e0f8a2a-e71f-43c8-b60e-50f0f34e6d69';
+----------------------------------------+-------------------+
|config                                  |created_at         |
+----------------------------------------+-------------------+
|{"hashed_password": "$legacyfandom$..."}|2024-02-02 10:03:36|
+----------------------------------------+-------------------+
```